sdk/include/opentelemetry/sdk/logs/logger.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "opentelemetry/logs/log_record.h"
+#include "opentelemetry/logs/logger.h"
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/sdk/common/atomic_shared_ptr.h"
+#include "opentelemetry/sdk/logs/processor.h"
+
+#include <unordered_map>
+#include <vector>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace logs
+{
+class Logger final : public opentelemetry::logs::Logger, public std::enable_shared_from_this<Logger>
+{
+public:
+  /**
+   * Initialize a new logger with a specified name.
+   * @param name The name of this logger.
+   */
+  explicit Logger() noexcept;
+
+  /**
+   * Initialize a new logger with a specified name and processor.
+   * @param name The name of this logger.
+   * @param processor The name of the processor.
```

Comment:
This is the processor instead of _the name of processor_?

Reply:
Yup good catch, I'll change that comment!